app/src/main/java/com/example/peterchu/watplanner/Models/Schedule/CourseComponent.java

```diff
@@ -281,15 +281,15 @@ public Calendar getCalendarEndTime() {
 
                     event.setStartTime(eventStartDate);
                     event.setEndTime(eventEndDate);
-                    event.setLocation(location.toString());
+                    event.setLocation(type);
```

Comment:
I just changed this because the strings always truncate and was tryna figure out a good, minimal set of data to display. I agree with you @timothytong that we should show section... Not sure about location though.